Serializer/Accessor/DefaultAccessorStrategy.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/*
+ * (c) 2018, OpticsPlanet, Inc.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace Mango\Bundle\JsonApiBundle\Serializer\Accessor;
+
+use JMS\Serializer\Accessor\DefaultAccessorStrategy as BaseAccessor;
+use JMS\Serializer\Metadata\PropertyMetadata;
+use Mango\Bundle\JsonApiBundle\Util\Model\AffectedPropertiesAwareInterface;
+
+/**
+ * DefaultAccessorStrategy is an override for original DefaultAccessorStrategy to add
+ * custom behavior like track affected properties, etc.
+ *
+ * @author Vlad Yarus <vladislav.yarus@intexsys.lv>
+ */
+class DefaultAccessorStrategy extends BaseAccessor
```

Comment:
@vlady777  you are able to cover this functionality with functional tests. Please see \Mango\Bundle\JsonApiBundle\Tests\Serializer\DeserializationTest

Reply:
Ticket was created for that. See related tickets.